eslintrc.js

```diff
@@ -71,5 +72,6 @@ module.exports = {
     "prefer-const": "error",
     "sort-requires/sort-requires": "error",
     "strict": ["error", "global"],
+    "eslint-comments/no-unused-disable": "error",
```

Comment:
[`eslint-comments/no-unused-disable`](https://mysticatea.github.io/eslint-plugin-eslint-comments/rules/no-unused-disable.html) rule does not be included in [`plugin:eslint-comments/recommended`](https://github.com/mysticatea/eslint-plugin-eslint-comments/blob/c427c1c0577fab56f2ff0919053d249938507c83/lib/configs/recommended.js) configuration, but I think it is useful.